students/john_navitsky/session01/warmup1/monkey_trouble.py

```diff
@@ -0,0 +1,13 @@
+
+def monkey_trouble(a_smile=False, b_smile=False):
+
+    bool_monkey=a_smile + b_smile
+    if bool_monkey in [2, 0]:
```

Comment:
slick! 

I wouldn't call the variable "bool_monkey", as it's not really a bool anymore. But I like the logic!